### ISSUE TITLE
assetic:dump causes RuntimeException (signal "6")

### ISSUE THREAD

**Issue by stevenmusumeche** [kriswallsmith/assetic#719]
I'm trying to dump my assets on a virtual machine with 8GB of memory like this:

```
php ./app/console assetic:dump --verbose -e prod
```

I get the following error when it gets to one of my larger scss files:

```
[Symfony\Component\Process\Exception\RuntimeException]
  The process has been signaled with signal "6".
```

If I run this in the dev environment, it works:

```
php ./app/console assetic:dump --verbose -e dev
```

Is it running into some sort of memory limit? How I can I allocate more memory to the dump process? I am using compass, uglifycss, and uglifyjs.

**Comment by gjuric** [comment]
The runtime exception indicates that the signal "6" was received from the external script you were using. So I don't think this is an assetic issue, but the runtime you use to process scss files issue.

If you take a look at Assetic\Filter\Sass\ScssFilter you can see that it is using Symfonys ProcessBulder to run the scss process, so you can examine the exact arguments being passed to the sass script and check what is different in your dev and prod environments.

**Comment by stevenmusumeche** [comment]
@gjuric  thank you for your response. The exception trace shows this exception is being thrown in the UglifyCssFilter class. It only happens when the files are being combined into one big file (production envirornment) instead of many files (dev environment).  This doesn't occur with the same code on other servers, that's why I am wondering if it is a memory issue.  Any other ideas on how to debug this?

**Comment by stevenmusumeche** [comment]
I was able to find the command that was running:

    '/usr/local/bin/node' '/usr/bin/uglifycss' '/tmp/inputhmo4tj'

The error is immediately 

    FATAL ERROR: CALL_AND_RETRY_LAST Allocation failed - process out of memory
    Aborted (core dumped)

**Comment by gjuric** [comment]
It is a node / uglifycss issue, so I would try to ask there. The only thing I found that you could try is to pass extra parameters regarding the memory available to the node process, see here for details -> http://stackoverflow.com/a/26094800

**Comment by xabbuh** [comment]
@stevenmusumeche Can this be closed given that fmarcia/UglifyCSS#25 has been resolved?

**Comment by stevenmusumeche** [comment]
Yes sorry
